test/suite/test_oligarch09.py

```diff
@@ -51,6 +51,7 @@ def conn_extensions(self, extlist):
         DisaggConfigMixin.conn_extensions(self, extlist)
 
     def test_oligarch_read_write(self):
```

Comment:
This still fails?

Reply:
Yes, I think it fails inconsistently.  Like if you just run this test on its own, it may pass, but if you run it with other tests, or in the context of a PR, it's much more likely to fail.